### DISCUSSION TITLE
Faire converger le plugin avec l'extension Menu from Project

### DISCUSSION THREAD

**Discussion by bchartier** [geo2france/idg-qgis3-plugin#4] [Ideas]
Il existe une similitude entre les objectifs de notre plugin et "Menu from Project" (voir https://github.com/xcaeag/MenuFromProject-Qgis-Plugin).

Je me pose donc la question de l'intérêt de faire vivre séparément ces 2 plugins.
Si vous avez un avis sur la question de faire converger ces deux outils ou non, n'hésitez pas à le donner ici.

**Comment by Doctor-Who** [comment]
Intéressant effectivement, l'approche fonctionnelle est la même mais l'ergonomie est différente.
@vfabry @bchartier aux GeoDataDays nous allons croiser au moins 2 des développeurs de ce plugin, à voir avec eux qui le finance, la roadmap, etc. Pour voir si le rapprochement est pertinent ? Qu'en pensez vous ?

**Reply by bchartier** [reply]
> qui le finance

et qui le réalise...

**Comment by Doctor-Who** [comment]
D'un point de vue fonctionnel, le plugin Menu from Project est intéressant.
Toutefois sur le plan ergonomie, je trouve la vue arborescente et le glisser/déposer plus facile d'utilisation de l'actuel plugin IDG que le système de menu hiérarchique du plugin Menu.
Je me posais même la pertinence de basculer ces éléments dans le panneau Explorateur.

**Comment by bchartier** [comment]
@Doctor-Who a contacté l'agence de l'eau Adour Garonne. Si j'ai bien compris la retranscription de l'échange, l'agence n'a pas d'attente particulière et n'a pas besoin de faire évoluer son plugin. Si ça nous arrange on peut faire quelque chose mais cela nous regarde.

**Comment by jbdesbas** [comment]
@bchartier et @jbdesbas avons participé à des ateliers sur LMFP sur invitation de J. Moura (merci !)

Bien que les utilisateurs finaux et le mode d'utilisation des plugins diffèrent, nous pouvons tout de même rester vigilant sur l'inter-compatibilité (au moins basique) des projets QGIS entre les 2 plateformes.
Aujourd'hui l'arborescence des couches fonctionne de la même manière entre IDG et LMFP.
A suivre pour les fonctionnalité avancées : millésimes, couches "stars", etc.
